RELEASING.md

```diff
@@ -75,3 +75,27 @@ The following steps must be completed by the primary on-call maintainer to perfo
 
 * PRs to update spiffe.io and spire-examples repo to the latest major version must be merged.
   * Ensure that the PRs have been updated to use the version tag instead of the commit sha.
+
+### SDK Releases
+
+SPIRE has two SDK repositories:
+
+* [API SDK](https://github.com/spiffe/spire-api-sdk)
+* [Plugin SDK](https://github.com/spiffe/spire-plugin-sdk)
+
+SPIRE consumes these SDKs using pseudo-versions from the `next` branch in each SDK repository. This allows unreleased changes to be reviewed, merged, and consumed by SPIRE.
+
+These SDKs need to be released with each SPIRE release.
+
+SDKS releases take place using tagged commits from the `main` branch in each repository. When cutting a new release, the `main` branch needs to be prepared with any previously unreleased changes that are part of the new release.
+
+To create a release for an SDK, perform the following steps:
+
+1. Review the diff between `next` and `main`.
+1. Determine what commits in `next` that are missing from `main` that should be included in the release.
```

Comment:
I'm wondering if we could include some details here that can help to aid this determination? E.g. What's the criteria to determine what are the commits that need to be cherry-picked. I'm trying to disambiguate the case where we picked a commit for SPIRE and then we added commits to the next branch in the SDKs that will not be included in that release. Maybe adding something like "... that should be included in the release because the matching SPIRE release depends on those commits." is enough?

Reply:
Sounds good. Took a stab at some new text.